python/lsst/summit/utils/efdUtils.py

```diff
@@ -452,22 +454,22 @@ def makeEfdClient(testing: bool | None = False) -> EfdClient:
         raise RuntimeError("Could not create EFD client because importing lsst_efd_client failed.")
 
     if testing:
-        return EfdClient("usdf_efd")
```

Comment:
I might have defined a dict with new args (just db_name for now), so that if you ever add another arg to pass through you won't have to modify all these lines again